File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.java
class Solution {
    public boolean isSortedCol(int col, String[] strs) {
        for(int i=1; i<strs.length; i++) {
            if(strs[i].charAt(col) < strs[i-1].charAt(col)) {
                return false;
            }
        }
        return true;
    }

    public int minDeletionSize(String[] strs) {
        int cnt = 0;
        for(int i=0; i<strs[0].length(); i++) {
            if(!isSortedCol(i, strs)) {
                cnt++;
            }
        }
        return cnt;
    }
}

File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        cnt = 0
        for col in range(len(strs[0])):
            for row in range(1, len(strs)):
                if strs[row][col] < strs[row - 1][col]:
                    cnt += 1
                    break

        return cnt
